app/views/children/_form.html.arb

```diff
@@ -39,12 +39,19 @@ context.instance_exec do
       end
 
       render 'children/form/childcare', form: self
+      if child.age <= 5
+        render 'children/form/childcare_medical_history', form: self
+      end
     end
 
     column do
       render 'people/form/cost_adjustments', form: self
       render 'people/form/stays', form: self
       render 'people/form/meal_exemptions', form: self, person: child
+      if child.age > 5
+        render 'children/form/cru_student_medical_history_misc', form: self
+        render 'children/form/cru_student_medical_history', form: self
```

Comment:
I think there is an issue with the form that applies to both childcare_medical_history and cru_student_medical_history: it's that if the child does not already have one of these records then they won't be displayed on the form, and then the admin has no way of editing the values.

Reply:
Ok, Changes have been made.  Now if you do not have medical history an option will be given to create some :)